noxfile.py

```diff
@@ -253,7 +253,11 @@ def create_zipapp(session: Session) -> Path:
 def test(session: Session) -> None:
     science_pyz = create_zipapp(session)
     test_env = {"BUILD_ROOT": str(BUILD_ROOT), "SCIENCE_TEST_PYZ_PATH": str(science_pyz)}
+
+    # TODO(John Sirois): Figure out why this fails for GH Windows 2022 runners and re-enable for
+    #  Windows: https://github.com/a-scie/lift/issues/9
     xdist_args = [] if IS_WINDOWS else ["-n" "auto"]
```

Comment:
It would be good to get to the bottom of this: https://github.com/a-scie/lift/issues/9
Unfortunately I can't repro on my Windows machine :/ I could using ptex 0.6.0, but the ptex 0.7.0 update fixed.